crmsh/config.py

```diff
@@ -294,7 +294,8 @@ def get(self, value):
         'collect_extra_logs': opt_string('/var/log/messages /var/log/pacemaker/pacemaker.log /var/log/pacemaker.log /var/log/crmsh/crmsh.log'),
```

Comment:
Would be useful to collect `crm.conf` and `profiles.yml` from all nodes too

Reply:
Added